config.md

```diff
@@ -265,6 +265,27 @@ Annotations are key-value maps.
 }
 ```
 
+## HypervisorPath
```

Comment:
On Fri, Apr 29, 2016 at 02:01:45AM -0700, James Hunt wrote:
“1) which runtime to use ("HypervisorPath" here, but should probably
be renamed "RuntimePath").”

This approach assumes a shim like containerd between the config author
and the runtime.  I'm fine with settings like this being part of the
contract between Docker and containerd, but if they are, containerd
should remove the setting before passing the config down to the
runtime.  This repo is about the runtime ↔ caller relationship, and in
that context “which runtime should I use?” has already been decided.

“Currently, the OCI bundle includes config.json, but does that imply
the file is "read-only"? Or will "OCI managers" (is there an official
term?) like docker be able to supplement (in other words rewrite) this
configuration based on locally available resources?  If so, what are
they allowed to change in the file?  Can they only supplement, or are
they permitted to change anything and everything?”

This spec is just about the final bundle passed to the runtime, so
this seems out of scope here.  I recommend “OCI container as a
service” providers clearly document their policies (e.g. “we clobber
‘hooks’ for safety”), but not in this repo ;).

“To me, this implies it might be better to have a "static"
(effectively read-only) config file…”

We tried this for a time.  See #284 for why we went back to a single
config.